cookiebar.info.yml

```diff
@@ -1,9 +1,9 @@
-name: KeesTM Cookiebar
+name: Cookiebar
 description: Add a cookies bar block to your site
-package: KeesTM
+package: estdigital
 type: module
-version: 2.0.0
-core: 8.x
```

Comment:
'core' kan denk ik wel blijven staan.
Als je dat weg haalt is de module niet meet compatible met versies lager dan 8.7.7 maar dat is de module wel lijkt me.
info:
https://www.drupal.org/docs/creating-custom-modules/let-drupal-know-about-your-module-with-an-infoyml-file